pkg/experiment/remote/client.go

```diff
@@ -60,19 +61,25 @@ func (c *Client) Fetch(user *experiment.User) (map[string]experiment.Variant, er
 // FetchV2 fetches variants for a user from the remote evaluation service.
 // Unlike Fetch, this method returns all variants, including default variants.
 func (c *Client) FetchV2(user *experiment.User) (map[string]experiment.Variant, error) {
-	variants, err := c.doFetch(user, c.config.FetchTimeout)
+	ctx := context.Background()
+	return c.FetchV2WithContext(ctx, user)
+}
+
+// FetchV2WithContext fetches variants for a user from the remote evaluation service with a context.
+func (c *Client) FetchV2WithContext(ctx context.Context, user *experiment.User) (map[string]experiment.Variant, error) {
```

Comment:
nitpick - can you re-order these function params: (user, ctx) rather than (ctx, user)

Reply:
Sure!
I put `ctx` first because I thought it was the "idiomatic" golang way. But the new order matches better with the other methods parameters.